packages/graphql-client/tsconfig.json

```diff
@@ -0,0 +1,16 @@
+{
+  "extends": "../../tsconfig",
+  "compilerOptions": {
```

Comment:
Some of these options are already defined in `../../tsconfig` do they need to be repeated here?